### ISSUE TITLE
RGB image incompatibility with firmware 16.23

### ISSUE THREAD

**Issue by lola-masson** [ifm/ifm3d-ros#67]
There is a bug in ifm3d-ros used in combination with the O3R firmware version 0.16.23 which makes it impossible to get RGB data.

**Comment by samialperen** [comment]
Indeed, confirmed this bug with ifm3d 0.93.0 and ifm3d-ros (noetic) 1.0.1
